src/man/sss_ssh_knownhosts.1.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE reference PUBLIC "-//OASIS//DTD DocBook V4.4//EN"
+"http://www.oasis-open.org/docbook/xml/4.4/docbookx.dtd">
```

Comment:
4027930598457940e750ec08a27c44bb718e279b bumped this to 4.5

Reply:
Fixed